Common/Core/FFitWeights.h

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/*
+Author: Joachim Hansen
+
+*/
```

Comment:
Please add documentation.

Reply:
Hi @vkucera,
I've added a short description in the header file for what it currently holds and what it will hold in the future. the source file refers to the header file now

Reply:
Thanks. Can you please format it using the Doxygen fields `\file`, `\author`, `\brief`, as described at https://rawgit.com/AliceO2Group/CodingGuidelines/master/comments_guidelines.html?showone=Files#Files ?

Reply:
@vkucera This should be done now :)

Reply:
Thanks